script.js

```diff
@@ -0,0 +1,148 @@
+const humburger = document.querySelector('.humburger');
+const x = document.querySelector('#exit');
+const popupList = document.querySelector('.nav ul');
+const secHeader = document.querySelector('.sec-header');
+const lis = document.querySelectorAll('.nav ul li');
+
+secHeader.classList.add('a-class');
+
+const close = function () {
+  x.classList.add('nullifier');
+  humburger.classList.remove('nullifier');
+  secHeader.classList.replace('popup', 'a-class');
+  popupList.classList.replace('popup', 'toggle');
+};
+
+const popUp = function () {
+  secHeader.classList.replace('a-class', 'popup');
+  humburger.classList.add('nullifier');
+  x.classList.remove('nullifier');
+  popupList.classList.add('popup');
+  popupList.classList.remove('toggle');
+
+  x.addEventListener('click', close);
+
+  lis.forEach((li) => li.addEventListener('click', close));
+};
+
+humburger.addEventListener('click', popUp);
+
+const features = document.querySelector('#introduction');
+// const activities = document.querySelector('.featured-activities');
```

Comment:
Great design @Yordinia 🤝

- [ ] You may always remove commented code from your project as this is regarded as a bad practice.